src/layout/NavigationLink.js

```diff
@@ -7,17 +7,41 @@ import { Link } from 'react-router';
 export class NavigationLink extends React.Component {
   static propTypes = {
     className: PropTypes.string,
-    to: PropTypes.string.isRequired
+    to: PropTypes.string,
+    tag: PropTypes.oneOf(['a', 'Link'])
```

Comment:
As discussed yesterday morning, let's not use tag as props. Simply act on if there is a props `to` or `href`. The usage would be:

```javascript
<NavigationLink to='/admin'>Admin</NavigationLink>
// or
<NavigationLink href='/other/admin'> Other Admin</NavigationLink>
```